scripts/waifulib/glslc.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# encoding: utf-8
+# a1batross, 2020
+
+import os
+from waflib import *
+from waflib.Tools import c_preproc
+
+def options(opt):
+	pass
```

Comment:
Implement

Reply:
No.

Reply:
Then remove.

Reply:
Then read https://waf.io/book and find out why it's left as stub.